src/libs/ReportUtils.ts

```diff
@@ -4449,6 +4449,14 @@ function getChatByParticipants(newParticipantList: number[], reports: OnyxCollec
     );
 }
 
+function getChatByReportID(reportID: string, reports: OnyxCollection<Report> = allReports) {
+    return (
+        Object.values(reports ?? {}).find((report) => {
+            return report?.reportID === reportID;
+        }) ?? null
```

Comment:
I am very confused by this... I think it's doing nothing? ie: if it is null then default to null